<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QPushButton{
    background:rgba(247,215,71,255);
    /*border: 1px solid #2361a0; /* Cambiar el color del borde a azul oscuro */
    border-radius: 10px;
    font: 75 12pt &quot;Archivo&quot;;
/* Cambiar el color de fondo a azul oscuro */
    color: rgba(128,96,0,255);
}
/*
QPushButton:hover{

background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(14, 52, 83, 1), stop:1 rgba(14, 52, 83, 1));
color: black;

}
*/
QGroupBox{
background-color: rgb(255, 255, 255); /* Cambiar el color de fondo a azul oscuro */
}

QLineEdit{
background-color: rgb(255,255,255);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
   /*color: #2361a0;*/
	color:rgba(106,105,103,255);
    font: 75 12pt &quot;Archivo&quot;;

}

QLabel{
color: rgba(106,105,103,255);
    font: 75 10pt &quot;Archivo&quot;;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="btn_cerrar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>630</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_guardar">
       <property name="geometry">
        <rect>
         <x>960</x>
         <y>630</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(102,56,178,255);
color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>10</y>
         <width>1151</width>
         <height>261</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Archivo</family>
          <pointsize>22</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgba(44,44,44,255);
    font: 75 22pt &quot;Archivo&quot;;
}</string>
        </property>
        <property name="text">
         <string>Proceso de salida</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_numeroHabitacion">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero habitacion</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_categoria">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Categoria</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_detalle">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>331</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Detalles</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_condicion">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Condicion</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_precio">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Precio</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_Cliente">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>50</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre Del Cliente:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_DPI">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>100</y>
          <width>341</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dpi</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_FechaEntrada">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>50</y>
          <width>261</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha de Entrada</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_Razon">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>150</y>
          <width>361</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Razon de Visita</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>1101</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Archivo</family>
          <pointsize>10</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgba(44,44,44,255);
    font: 75 10pt &quot;Archivo&quot;;
}</string>
        </property>
        <property name="text">
         <string>Costo de alojamiento </string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tablaAlojamiento">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>1071</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView {
    background-color: white;
    border: none;
    font: 75 12pt &quot;Archivo&quot;;
}

QHeaderView:hover {
    background-color: white;
}

QHeaderView:section {
    background-color: white;
    border-bottom: 1px solid lightgray;
    border-top: none;
    padding: 2px;
    font-size: 11pt;
    color: black;
text-align: left;
}

QTableWidget {
    background-color: white;
    border: none;
    gridline-color: transparent;
    font-size: 11pt;
    color: black;

}

QTableWidget QTableCornerButton::section {
    background-color: white;
    border: none;

}

QTableWidget::item {
    border: none;
    border-bottom: 1px solid lightgray;
    alignment: AlignHCenter; /* Centra el texto horizontalmente */
	color:black /* cambiar el color letra al seleccionar*/

}</string>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>175</number>
        </attribute>
        <column>
         <property name="text">
          <string>Costo Calculado </string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dinero Pagado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>/</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="tablaExtras">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>1071</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView {
    background-color: white;
    border: none;
    font: 75 12pt &quot;Archivo&quot;;
}

QHeaderView:hover {
    background-color: white;
}

QHeaderView:section {
    background-color: white;
    border-bottom: 1px solid lightgray;
    border-top: none;
    padding: 2px;
    font-size: 11pt;
    color: black;
text-align: left;
}

QTableWidget {
    background-color: white;
    border: none;
    gridline-color: transparent;
    font-size: 11pt;
    color: black;

}

QTableWidget QTableCornerButton::section {
    background-color: white;
    border: none;

}

QTableWidget::item {
    border: none;
    border-bottom: 1px solid lightgray;
    alignment: AlignHCenter; /* Centra el texto horizontalmente */
	color:black /* cambiar el color letra al seleccionar*/

}</string>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>175</number>
        </attribute>
        <column>
         <property name="text">
          <string>Extra</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Precio</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>/</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Archivo</family>
          <pointsize>10</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgba(44,44,44,255);
    font: 75 10pt &quot;Archivo&quot;;
}</string>
        </property>
        <property name="text">
         <string>Extras</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_Total">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>270</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Archivo</family>
          <pointsize>9</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgba(44,44,44,255);
    font: 75 9pt &quot;Archivo&quot;;
}</string>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
